cmd/completion.go

```diff
@@ -1,72 +0,0 @@
-package cmd
```

Comment:
Was this not previously used ? I don't see how this is related to the other non-installation bits that you removed. It feels more like a general shell completion feature.

Reply:
I actually have no idea. I never used it with original client.

Reply:
That's what I'm saying: I think we should keep it. It can apparently be used to configure shell completion for several shell flavours.

Reply:
OK, let me try to bring it back...